chart/templates/daemonset-nvidia-driver-installer.yaml

```diff
@@ -17,7 +17,9 @@ spec:
       - name: installer
         image: {{ .Values.nvidia.image }}
         securityContext:
-          privileged: true
+          capabilities:
+            add:
+              - SYS_ADMIN
```

Comment:
FWIW, the GPU Operator DaemonSets are running as privileged. i.e. https://github.com/NVIDIA/gpu-operator/blob/main/manifests/state-driver/0500_daemonset.yaml#L166-L167

But if SYS_ADMIN is less than privileged and still gets the job done then I agree we should go with it.

Reply:
Yes, the drivers installation and nvidia-smi worked with just SYS_ADMIN in my testing for both SLES and SLE Micro.